plugins/gitignore/gitignore.plugin.zsh

```diff
@@ -1,4 +1,4 @@
-function gi() { curl -sL https://www.gitignore.io/api/$@ ;}
+function gi() { curl -sL https://www.gitignore.io/api/`echo $@ | sed s/" "/,/g` ;}
```

Comment:
A better way to join arguments with a delimiter is with zsh builtin join syntax: `${(j:delimiter:)variable}`. In this case:
```zsh
curl -sL https://www.gitignore.io/api/${(j:,:)@}
```